docs/rfcs/UserProfilesv2.json

```diff
@@ -1,276 +1,435 @@
-                }]
-
-	},
-	"PGPFingerprints": {
-		"metadata": {
-			"authority": "mozilliansorg",
-			"signature": "ZOWSLXKxx..",
-			"classification": "PUBLIC"
-		},
-		"value": [{
-		            "value": "-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nmQINBE94eWwBEADjlvvF8HERvp.....=A0dq\n-----END PGP PUBLIC KEY BLOCK-----\n",
-                            "name": "moz key",
-                            "verified": false,
-                            "verifier": null,
-                            "primary": false,
-                            "lastModified": "2017-03-09T21:28:51.851Z"
-                }]
-	},
-	"picture": {
-		"metadata": {
-			"authority": "mozilliansorg",
-			"signature": "ZOWSLXKxx..",
-			"classification": "PUBLIC"
-		},
-		"value": "https://s.gravatar.com/avatar/ec6e85d15f8411d32f97f5d8a4eab2d3?s=480&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Flm.png"
-	},
-	"accessInformation": {
-		"ldap": {
-			"metadata": {
-				"authority": "ldap",
-				"signature": "ZOWSLXKxx..",
-				"classification": "PUBLIC"
-			},
-			"value": [{
-				        "created": "2010-01-23T04:56:22Z",
-				        "lastUsed": "2010-01-23T04:56:22Z",
-					"name": "vpn_default",
-                                        "value": null
-                        }]
-		},
-		"mozilliansorg": {
-			"metadata": {
-				"authority": "mozilliansorg",
-				"signature": "ZOWSLXKxx..",
-				"classification": "PUBLIC"
-			},
-			"value": [{
-				        "created": "2010-01-23T04:56:22Z",
-				        "lastUsed": "2010-01-23T04:56:22Z",
-					"name": "nda",
-                                        "value": null
-                        }]
-		},
-		"hris": {
-			"metadata": {
-				"authority": "hris",
-				"signature": "ZOWSLXKxx..",
-				"classification": "STAFF CONFIDENTIAL"
-			},
-			"value": [{
-					"name": "costcenter",
-					"value": "1420"
-				},
-				{
-					"name": "workertype",
-					"value": "employee"
-				},
-				{
-					"name": "egencia",
-					"value": "uk"
-				},
-				{
-				        "created": "2010-01-23T04:56:22Z",
-				        "lastUsed": "2010-01-23T04:56:22Z",
-					"name": "ManagerName",
-					"value": "Mark"
-				}
-			]
-		},
-		"auth0": {
-			"metadata": {
-				"authority": "auth0",
-				"signature": "ZOWSLXKxx..",
-				"classification": "STAFF CONFIDENTIAL"
-			},
-			"value": [{
-				"created": "2010-01-23T04:56:22Z",
-				"lastUsed": "2010-01-23T04:56:22Z",
-				"name": "mozdef1.private.scl3.gmail",
-				"value": "5a5munnfxYjqkaN0su1Kl7USxbqkILQN"
-			}]
-		}
-	},
-	"experimental": {
-		"metadata": {
-			"authority": "mozilliansorg",
-			"signature": "ZOWSLXKxx..",
-			"classification": "WORKGROUND CONFIDENTIAL"
-		},
-		"value": null
-	}
+  "_schema": "https://person-api.sso.mozilla.com/schema/v2/profile",
+  "user_id": {
+    "metadata": {
+      "publisher": "auth0",
+      "signatures": [
+        {
+          "name": "ZOWSLXKxx..",
+          "value": "LALALA...",
+            "type": "jwks"
```

Comment:
nit : All the `type` field indentations are off (maybe tabs vs spaces)

Reply:
maybe we should convert to YAML.  In theory the standard supports it.